main.tf

```diff
@@ -104,16 +104,12 @@ resource "google_compute_firewall" "login_to_vm" {
   count         = (local.create_firewalls && var.allow_login) ? 1 : 0
   name          = local.vm_login_firewall_name
   network       = google_compute_instance.vm_instance.network_interface.0.network
-  source_ranges = [local.google_iap_cidr /* see https://stackoverflow.com/a/57024714/636762 */]
+  source_ranges = concat([local.google_iap_cidr /* see https://stackoverflow.com/a/57024714/636762 */], distinct(var.fw_allowed_cidrs))
   target_tags   = local.network_tags
   depends_on    = [google_project_service.networking_api]
   allow {
     protocol = "tcp"
-    ports = [
-      # https://cloud.google.com/iap/docs/using-tcp-forwarding#create-firewall-rule
-      22,   # for SSH
-      3389, # for RDP
-    ]
+    ports    = concat(["22", "3389"], distinct(var.fw_allowed_ports))
```

Comment:
Please don't get rid of those helpful comments. They have done.a GREAT JOB at helping new developers understand what these numbers are why they are here 🙂

FInd a way to `distinct()` `concat()` while keeping these helpful comments where they belong.

Reply:
Done
Used:
```
ports = distinct(
      concat(
        [
          "22",  # ssh
          "3389" # rdp
        ],
        (var.fw_allowed_ports)
    ))
```